src/c++/perf_analyzer/concurrency_worker.cc

```diff
@@ -105,6 +101,13 @@ ConcurrencyWorker::HandleExecuteOff()
       thread_config_->is_paused_ = true;
       std::unique_lock<std::mutex> lock(wake_mutex_);
       wake_signal_.wait(lock, [this]() { return early_exit || execute_; });
+
+      // Unpause all sequences
```

Comment:
Why this was added: Only one of the two classes needed pausing and unpausing of sequences, but to make them both be able to use the same Send Infer and CompleteSeq code, they both needed to be aware of sequence pausing

Reply:
This feels like a function to me. If you can have the comment above, why not make this `UnpauseAllSequences()`

Reply:
Talked offline. We are going to defer this to a later stage in the refactor work.